contributing/release-schedule.md

```diff
@@ -6,8 +6,8 @@
 - [Determining if your change is in a release](#determining-if-your-change-is-in-a-release)
 - [Release Cadence](#release-cadence)
   - [Detailed schedule](#detailed-schedule)
-  - [AMP Dev Channel (Experimental)](#amp-dev-channel-experimental)
-  - [AMP Beta Channel (Beta)](#amp-beta-channel-beta)
+  - [AMP Experimental Channel](#amp-experimental-channel)
+  - [AMP Beta Channel](#amp-beta-channel)
```

Comment:
Note: @mrjoro suggested merging these sections since they are rather similar and differ only in the release provided. @danielrozenberg WDYT?

Reply:
No objection from me on this one.

Reply:
Done.